core/templates/dev/head/components/exploration_summary_tile_directive.html

```diff
@@ -79,21 +113,79 @@ <h2 class="exploration-title protractor-test-exp-summary-tile-title"><[getExplor
 
   .oppia-exploration-summary-tile .top-section {
     height: 144px;
-    opacity: 1.0;
     position: relative;
     width: 100%;
   }
 
-  .oppia-exploration-summary-tile:hover .top-section {
-    opacity: 0.8;
-  }
-
   .oppia-exploration-summary-tile,
   .oppia-exploration-summary-tile .top-section {
     -webkit-transition: all 300ms;
     transition: all 300ms;
   }
 
+  .exploration-summary-contributors {
```

Comment:
Prefix with .oppia-exploration-summary-tile, otherwise this applies globally. Ditto elsewhere.